test/test_0_basic.py

```diff
@@ -1,21 +1,34 @@
-import os
+import textwrap
 import platform
+from .template_projects import CTemplateProject
+from . import utils
 
-import utils
+basic_project = CTemplateProject(
+    setup_py_add=textwrap.dedent('''
```

Comment:
I've tried to avoid it in a few cases (setting file contents as a dict assignment, for example), but I think it's unavoidable in this case. I don't think I want to auto-dedent as magic like that might just increase confusion for a first-time reader. I think it's something we'll have to live with! FWIW, I've gotten completely used to them - my eyes barely register them when I'm reading.